src/main/java/com/atlassian/db/replica/spi/circuitbreaker/CircuitBreaker.java

```diff
@@ -1,9 +1,18 @@
 package com.atlassian.db.replica.spi.circuitbreaker;
 
-import com.atlassian.db.replica.api.circuitbreaker.BreakerState;
```

Comment:
`CircuitBreaker` is the only member of `spi.circuitbreaker`. The `api.circuitbreaker` doesn't even exist. Shall I move it to `spi`?